src/store/mutations/mutations-model.ts

```diff
@@ -413,3 +413,11 @@ export const mutateRestorationType = (state: StateIF, type: RestorationTypes) =>
 export const mutateRestorationExpiry = (state: StateIF, expiry: string) => {
   state.stateModel.restoration.expiry = expiry
 }
+
+export const mutateRestorationRelationships = (state: StateIF, relationships: RelationshipTypes[]) => {
+  state.stateModel.restoration.relationships = relationships
+}
+
+export const mutateRestorationIsValid = (state: StateIF, isRestorationTypeValid: boolean) => {
```

Comment:
There's a mismatch between the mutation name and the thing you're modifying. Please fix.